README.md

```diff
@@ -58,3 +58,188 @@ select * where {
+	environment:
+		  CONNECT_BOOTSTRAP_SERVERS: SSL://your_host:your_port
+		  CONNECT_REST_ADVERTISED_HOST_NAME: your_host
+		  CONNECT_REST_PORT: port
+		  CONNECT_REST_HOST_NAME: host
+		  CONNECT_GROUP_ID: compose-connect-group-id
+		  CONNECT_KEY_CONVERTER: key_serializer.class
+		  CONNECT_VALUE_CONVERTER: value_serializer.class
+		  CONNECT_INTERNAL_KEY_CONVERTER: key_converter.class
+		  CONNECT_INTERNAL_VALUE_CONVERTER: value_converter.class
+		  CONNECT_CONFIG_STORAGE_TOPIC: docker-connect-configs-topic
+		  CONNECT_CONFIG_STORAGE_REPLICATION_FACTOR: 1
+		  CONNECT_OFFSET_FLUSH_INTERVAL_MS: 10000
+		  CONNECT_OFFSET_STORAGE_TOPIC: docker-connect-offsets-storage-topic
+		  CONNECT_OFFSET_STORAGE_REPLICATION_FACTOR: 1
+		  CONNECT_STATUS_STORAGE_TOPIC: docker-connect-status-storage-topic
+		  CONNECT_STATUS_STORAGE_REPLICATION_FACTOR: 1
+		  CONNECT_ZOOKEEPER_CONNECT: example.test.com:2181
+
+		  CONNECT_SASL_MECHANISM: PLAIN
+		  CONNECT_CONSUMER_SASL_MECHANISM: PLAIN
+		  CONNECT_PRODUCER_SASL_MECHANISM: PLAIN
+		  CONNECT_SASL_JAAS_CONFIG: |
+			org.apache.kafka.common.security.plain.PlainLoginModule required \
+			username="username" \
+			serviceName="servicename" \
+			password="password";
+		  CONNECT_CONSUMER_SASL_JAAS_CONFIG: |
+			org.apache.kafka.common.security.plain.PlainLoginModule required \
+			username="username" \
+			serviceName="servicename" \
+			password="password";
+		  CONNECT_PRODUCER_SASL_JAAS_CONFIG: |
+			org.apache.kafka.common.security.plain.PlainLoginModule required \
+			serviceName="servicename" \
+			username="username" \
+			password="password";
+		  CONNECT_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM: HTTPS
+
+		  CONNECT_SECURITY_PROTOCOL: SASL_SSL
+		  CONNECT_SSL_TRUSTSTORE_FILENAME: /path_to/keystore.jks
+		  CONNECT_SSL_TRUSTSTORE_LOCATION: /path_to/keystore.jks
+		  CONNECT_SSL_TRUSTSTORE_PASSWORD: changeit
+		  CONNECT_SSL_KEYSTORE_FILENAME: /path_to/keystore.jks
+		  CONNECT_SSL_KEYSTORE_LOCATION: /path_to/keystore.jks
+		  CONNECT_SSL_KEYSTORE_PASSWORD: changeit
+		  CONNECT_SSL_KEY_PASSWORD: my_password
+
+		  CONNECT_CONSUMER_SECURITY_PROTOCOL: SASL_SSL
+		  CONNECT_CONSUMER_SSL_TRUSTSTORE_LOCATION: /path_to/keystore.jks
+		  CONNECT_CONSUMER_SSL_TRUSTSTORE_PASSWORD: changeit
+		  CONNECT_CONSUMER_SSL_KEYSTORE_LOCATION: /path_to/keystore.jks
+		  CONNECT_CONSUMER_SSL_KEYSTORE_FILENAME: /path_to/keystore.jks
+		  CONNECT_CONSUMER_SSL_KEYSTORE_PASSWORD: changeit
+		  CONNECT_CONSUMER_SSL_KEY_PASSWORD: my_password
+
+		  CONNECT_PRODUCER_SECURITY_PROTOCOL: SASL_SSL
+		  CONNECT_PRODUCER_SSL_TRUSTSTORE_LOCATION: /path_to/keystore.jks
+		  CONNECT_PRODUCER_SSL_TRUSTSTORE_PASSWORD: changeit
+		  CONNECT_PRODUCER_SSL_KEYSTORE_LOCATION: /path_to/keystore.jks
+		  CONNECT_PRODUCER_SSL_KEYSTORE_FILENAME: /path_to/keystore.jks
+		  CONNECT_PRODUCER_SSL_KEYSTORE_PASSWORD: changeit
+		  CONNECT_PRODUCER_SSL_KEY_PASSWORD: my_password
+
+		  CONNECT_PRODUCER_SSL_KEYSTORE_CREDENTIALS: /path_to/kafka-cred.properties
```

Comment:
I am still confused why one and the same property has to be added multiple times. I am referring to `/path_to/kafka-cred.properties`.